File: ProyectoBaseWebApp/app/Espesor.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Espesor extends Model
{
    protected $fillable = [
        'empresa_id', 'descripcion', 'valor', 'color', 'estado', 'creador_id', 'modificador_id'
    ];
    public function scopeActive($query)
    {
        return $query->where('estado', 'A');
    }
    public function creador()
    {
        return $this->belongsTo('App\User');
    }
    public function modificador()
    {
        return $this->belongsTo('App\User');
    }
    public function scopeCurrent($query, $empresaId)
    {
        return $query->where('empresa_id', $empresaId);
    }
    protected $table = 'espesores';
}

File: CosechasApp/app/src/main/java/com/techtraining/cosechasapp/db/Espesor.java
package com.techtraining.cosechasapp.db;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;
@Entity
public class Espesor {
    @PrimaryKey
    public int id;

    @ColumnInfo(name = "descripcion")
    public String descripcion;

    @ColumnInfo(name = "valor")
    public double valor;

    @ColumnInfo(name = "color")
    public String color;

    @ColumnInfo(name = "empresa_id")
    public int empresaId;

    @ColumnInfo(name = "estado")
    public String estado;
}
